www/_posts/2020-06-01-tools-release.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Create, Common, and XCode Released!"
```

Comment:
```suggestion
title:  "Cordova Create, Common, and Xcode Released!"
```

My inner grammar nazi says: Apple styles it as `Xcode`

Reply:
Good catch, it **is** "Xcode", not "XCode"